src/download.test.ts

```diff
@@ -74,6 +76,16 @@ describe("getSkylinkUrl", () => {
 
     expect(url).toEqual(`${portalUrl}/skynet/skylink/${skylink}?attachment=true`);
   });
+
+  it("should convert base64 skylinks to base32", () => {
+    const expectedBase32 = "https://bg06v2tidkir84hg0s1s4t97jaeoaa1jse1svrad657u070c9calq4g.siasky.net";
+
+    validSkylinkVariations.forEach((input) => {
+      const url = client.getSkylinkUrl(input, { base32: true, subdomain: true });
```

Comment:
`base32` property should not be necessary here

```suggestion
      const url = client.getSkylinkUrl(input, { subdomain: true });
```